tests/pytorch/test_pytorch_model_export.py

```diff
@@ -440,6 +441,211 @@ def predict(self, context, model_input):
         decimal=4)
 
 
+def test_load_pyfunc_loads_torch_model_using_pickle_module_specified_at_save_time(
+        module_scoped_subclassed_model, model_path):
```

Comment:
I wonder if it would be better to add a pickle and unpickle methods to the test and use this module as a pickle module and verify the model was actually loaded via this module. mocking torch load model seems a bit indirect. What do you think?

Reply:
makes sense.